eesp-ikev2.org

```diff
@@ -326,11 +326,12 @@ ephemeral source ports. Instead, IKE traffic should be confined to
 the source and destination ports to ensure proper protocol operation
 and maintain compatibility with existing implementations.
 
-When using ephemeral source ports, the receiver can only set the
-source port upon arrival of an EESP packet with that Sub SA ID. If
-the receiver is pre-populating a Sub SA, it may have to install it
-with a source port set to zero and, upon arrival of a packet,
-update the source port using a mapping change.
+When using ephemeral source ports, for a Sub SA, the receiver can
+only set the source port upon arrival of an EESP packet with that
+Sub SA ID. If the receiver is pre-populating a Sub SA, it may have
+to install it with a source port set to zero and, upon arrival of a
+packet, update the source port using a mapping change for for this,
+uni-directional Sub SA
```

Comment:
Hm, how does this relate to SK_sub (as the title suggests)? And aren't all IPsec SAs unidirectional by definition? Why point it out here specifically?

Reply:
oh that is not part of this. That was part of UDP encap clarifications.I will move it there. Thanks.